nautobot/project-static/src/scss/nautobot.scss

```diff
@@ -428,7 +428,13 @@ $theme-colors-rgb-dark: map-loop($theme-colors-dark, to-rgb, "$value");
 /* 7. Optionally include utilities API last to generate classes based on the Sass map in `_utilities.scss` */
 @import "bootstrap/scss/utilities/api";
 
-/* 8. Add additional custom code here */
+/* 8. Include select2 and select2-bootstrap5-theme here */
+@import "select2/src/scss/core";
+// this is a workaround for select2-bootstrap5-theme issue https://github.com/apalfrey/select2-bootstrap-5-theme/issues/75
```

Comment:
I am concerned that the `select2-bootstrap5-theme` repo has been archived, suggesting it's unmaintained at this time. Good enough for now but we might need/want to see if there's any more up-to-date forks that we should switch to, or even whether we should maintain our own fork.